packages/labs/test-projects/test-element-a/package.json

```diff
@@ -36,6 +36,10 @@
         "build"
       ],
       "command": "npm pack",
+      "files": [
+        "package.json",
```

Comment:
.gitignore also? I believe `npm pack` etc reads that.

Reply:
It doesn't read `.gitignore`. Otherwise our `.js` artifacts wouldn't get included. It does read `.npmignore`, but we don't use those -- we consistently use the allowlist `files` approach instead.